crates/oxc_linter/src/rules/jest/no_standalone_expect/tests/jest.rs

```diff
@@ -74,6 +74,18 @@ fn test() {
             });",
             None,
         ),
+        (
```

Comment:
I think this is the same case as the one just added. Still doesn't hurt to have more.